test/controllers/authentications_controller_test.rb

```diff
@@ -52,4 +60,21 @@ class AuthenticationsControllerTest < ActionDispatch::IntegrationTest
     assert_response :no_content
     refute cookies[JWTSessions.access_cookie].present?
   end
+
+  test 'when logging out successfully the Session invalidated fields are updated' do
```

Comment:
While this new test only checks that the `Session` record has been set to invalidated and the invalidate by user is present.